src/rp2_common/pico_runtime/include/pico/runtime.h

```diff
@@ -49,13 +49,26 @@ void runtime_init(void);
 
 void runtime_run_initializers(void);
 void runtime_run_per_core_initializers(void);
+// PICO_CONFIG: PICO_RUNTIME_SKIP_INIT_ALL, Skip calling of `runtime_init_` functions during runtime init, type=bool, default=0, group=pico_runtime_init
```

Comment:
So presumably this "overrides" e.g. `PICO_RUNTIME_SKIP_INIT_POST_CLOCK_RESETS` ?